htdocs/comm/action/card.php

```diff
@@ -1114,7 +1114,8 @@ function init_repeat()
 			$events[]=array('method' => 'getContacts', 'url' => dol_buildpath('/core/ajax/contacts.php?showempty=1',1), 'htmlname' => 'contactid', 'params' => array('add-customer-contact' => 'disabled'));
 			// TODO Refresh also list of project if $conf->global->PROJECT_ALLOW_TO_LINK_FROM_OTHER_COMPANY not defined with list linked to socid ?
 			// FIXME If we change company, we may get a project that does not match
-			print $form->select_company($object->socid, 'socid', '', 'SelectThirdParty', 1, 0, $events, 0, 'minwidth200');
+			print $form->select_thirdparty_list($object->socid, 'socid', '', 'SelectThirdParty', 1, 0, $events);
```

Comment:
@eldy 
If I understand when we set "Wait you press a key before loading content of thirdparties combo list" to "No" the parameter $events can't be used ?
I was looking the code about "ajax_autocompleter" function but $events is never used and can't be used too, is it right?